src/view/com/composer/Composer.tsx

```diff
@@ -896,13 +901,15 @@ function ComposerPills({
             style={bottomBarAnimatedStyle}
           />
         )}
-        <LabelsBtn
-          labels={draft.labels}
-          onChange={nextLabels => {
-            dispatch({type: 'update_labels', labels: nextLabels})
-          }}
-          hasMedia={hasMedia || hasLink}
-        />
+        {containsMedia ? (
+          <LabelsBtn
+            labels={draft.labels}
+            onChange={nextLabels => {
+              dispatch({type: 'update_labels', labels: nextLabels})
+            }}
+            hasMedia={hasMedia || hasLink}
```

Comment:
i think this prop can be removed now? so then it would make sense to go into `LabelsBtn` and make all branches that depend on `hasMedia` truthy